src/scrape_up/github/organization.py

```diff
@@ -344,7 +344,25 @@ def pinned_repository(self):
 
         except:
             return "no pinned repository found for this organisation"
+    def get_organization_links(self):
+        try:
+            links={}
+            data=self.__scrape_page()
+            website_link=data.find("a",rel="nofollow",itemprop="url",href=True)['href']
+            links['website']=website_link
+            gmail=data.find("a",itemprop="email",href=True)['href']
+            links['gmail']=gmail
+            other_link=data.find_all("a",rel="nofollow",href=True)
+            for o in other_link:
+                name=o['href'].split("//")[1].split('/')[0].replace("www.","").split('.')[0]
+                if name!=self.organization or name.find(self.organization)==-1 :
+                    if not name in links:
```

Comment:
return the links in the try block only and instead of printing a message in the `except` block, return a message.

Reply:
Done!!